src/opscenarios.jl

```diff
@@ -372,8 +372,10 @@ _opscen(sro::StratReprOpscenario) = sro.opscen
 _strat_per(sro::StratReprOpscenario) = sro.sp
 _rper(sro::StratReprOpscenario) = sro.rp
 
+mult_repr(sro::StratReprOpscenario) = sro.mult_rp
```

Comment:
Should we export this function?

Reply:
We could. It could be useful in general when we want to know the exact value within an representative period. In EMX, we however use the directly scaled value.